packages/@webex/plugin-cc/src/services/AgentService.ts

```diff
@@ -56,4 +64,28 @@ export default class AgentService {
       return Promise.reject(error);
     }
   }
+
+  public async getBuddyAgents(options: BuddyAgents): Promise<BuddyAgentsSuccess> {
+    try {
+      const data = await this.httpRequest.sendRequestWithEvent({
```

Comment:
Suggestion (Not to be fixed in this PR) - We should allow `sendRequestWithEvent` to accept a response type so that we don't have to typecast the response while returning.

cc - @Kesari3008, Ravi